From e67cb7c04cacfdf98b07c4eedf10b5813c8e4deb Mon Sep 17 00:00:00 2001
From: Chris Wilson <chris@chris-wilson.co.uk>
Date: Thu, 12 Aug 2010 10:57:04 +0100
Patch-mainline: 2.6.37
References: fate#310916
Git-commit: 7340ea7dcf32227a77ed9df154fc2b2f00ad2fb1
Subject: [PATCH 1311/2588] drm/i915/overlay: Remove duplicated definition of
 OFC_UPDATE

Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
(cherry picked from commit 7340ea7dcf32227a77ed9df154fc2b2f00ad2fb1)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/i915/intel_overlay.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_overlay.c b/drivers/gpu/drm/i915/intel_overlay.c
index 3236cca..cc86b3e 100644
--- a/drivers/gpu/drm/i915/intel_overlay.c
+++ b/drivers/gpu/drm/i915/intel_overlay.c
@@ -170,9 +170,6 @@ struct overlay_registers {
     u16 RESERVEDG[0x100 / 2 - N_HORIZ_UV_TAPS * N_PHASES];
 };
 
-/* overlay flip addr flag */
-#define OFC_UPDATE		0x1
-
 static struct overlay_registers *
 intel_overlay_map_regs_atomic(struct intel_overlay *overlay,
 			      int slot)
-- 
1.7.6

